Src/Models/CBR.cpp

```diff
@@ -5,20 +5,26 @@
 
 #include <Misc/BinaryReader.h>
 
+#include "../Collision/CollisionMesh.h"
 #include "../Graphics/GraphicsResources.h"
 #include "../Utils/TextureUtil.h"
 
 const PGE::FilePath shaderPath = PGE::FilePath::fromStr("SCPCB/GFX/Shaders/RoomOpaque/");
 const PGE::FilePath shaderNormalPath = PGE::FilePath::fromStr("SCPCB/GFX/Shaders/RoomOpaqueNormalMap/");
-
 const PGE::String texturePath = "SCPCB/GFX/Map/Textures/";
 
+
 enum Lightmapped {
     No = 0,
     Fully = 1,
     Outdated = 2,
 };
 
+inline bool fileExists(PGE::String filename) {
```

Comment:
Don't use inline, it's the compiler's job to inline what it deems useful.

Reply:
Noted, thanks!
also removed for PGE::FilePath().exists()